code/modules/paperwork/prefab_papers/wey_yu/liaison/nda_short.dm

```diff
@@ -0,0 +1,142 @@
+	dat += "text-transform: uppercase;"
+	dat += "margin-top: 2px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-signature-small{"
+	dat += "font-size: 16px;"
+	dat += "font-style: italic;"
+	dat += "line-height: 1.4;"
+	dat += "margin-top: -6px;"
+	dat += "}"
+	dat += ""
+	dat += ".message-signature-s-small{"
+	dat += "font-size: 10px;"
+	dat += "font-style: italic;"
+	dat += "line-height: 1.4;"
+	dat += "margin-top: -20px;"
+	dat += "}"
+	dat += ""
+	dat += ".disclaimer{"
+	dat += "font-size: 8px;"
+	dat += "line-height: 1.2;"
+	dat += "}"
+	dat += ""
+	dat += "</style>"
+	dat += ""
+	dat += "<div id=\"width-container\">"
+	dat += "<div id=\"wy-fax-logo\">"
+	dat += "<img src='[asset.get_url_mappings()["logo_wy.png"]]' alt='WY Logo'/>"
+	dat += "</div>"
+	dat += ""
+	dat += "<!--Header Info here-->"
+	dat += "<div class=\"message-header-text\">"
+	dat += "<p class=\"header-title\">Confidentiality Agreement</p>"
+	dat += "<p class=\"header-subtitle\">WY441</p>"
+	dat += "<hr style=\"margin-top:5px; margin-bottom:35px;\">"
+	dat += "</div> <!-- /Heasder Info -->"
+	dat += ""
+	dat += ""
+	dat += "<!-- Message -->"
+	dat += "<div class=\"message-body-text\">"
+	dat += "<p>I, <span class=\"paper_field\"></span>, agree to a Confidentiality Agreement with the Weyland-Yutani (Space) Corporation that no information regarding the events that took place on the facility named <span class=\"paper_field\"></span> and subsequently the USS Almayer is to be delivered publicly. Only qualified Corporate personnel whose identities have been confirmed will receive such information.</p>"
+	dat += "<p>"
+	dat += "Signature: <span class=\"paper_field\"></span><br>"
+	dat += "Liaison Signature: <span class=\"paper_field\"></span><br>"
+	dat += "Date: <font face=\"Times New Roman\"><i>[time2text(REALTIMEOFDAY, "Day DD Month [GLOB.game_year]")]</i></font>"
+	dat += "</p>"
+	dat += "</div> <!-- /message-body-text -->"
+	dat += ""
+	dat += "<div class=\"message-signature-text\">"
+	dat += "<p class=\"disclaimer\">Our organization accepts no liability for the content of this missive, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. If you are not the intended recipient, you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.</p>"
+	dat += "<p class=\"disclaimer\">This message and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. This message contains confidential information and is intended only for addressee, if you are not the aforementioned addressee you should not disseminate, distribute or copy this missive. If you are not the intended recipient, you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.</p>"
```

Comment:
```only for the addressee. If you are not the aforementioned addressee, you should not disseminate```...
add article "the" before addressee. end sentence, start new instruction; comma after "aforementioned addressee", clause stip.